buildlogger.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package cedar;
 
-option go_package = "gopb";
+option go_package = "./gopb";
```

Comment:
Is this something that's required by the newer version of protobuf?

Reply:
Yeah, it's really annoying. They really don't want you to build your pb files in the same directory. But it requires either a slash or dot.